src/components/OrgChart.vue

```diff
@@ -182,10 +182,11 @@ export default {
 		background-color: var(--color-main-background);
 	}
 
-	::v-deep .node-button-div {
-		color: var(--color-primary-element);
-		div {
-			background-color: var(--color-main-background) !important;
+	.node-button-div {
+		filter: var(--background-invert-if-dark);
+
+		& > div > div {
```

Comment:
Dat selector tho :laughing:

Reply:
yeah I know. It is a shame that we are not in control over the library used...